### PR TITLE
style: :lipstick: make a landing page section, with illustration

### PR DESCRIPTION
**Pull request by lwjohnst86**
## Description

In the process of making a decision record for commit linting, I decided to update the landing page. See the screen shot below.

Screenshot from 2024-10-25 10-30-59

This PR needs a quick review.

## Checklist

- [x] Formatted Markdown
- [x] Rendered website locally

### CODE REVIEW

**Comment by K-Beicher** [comment]
It may just be me, but the heading Archive makes me think that there are also some posts that haven't been 'archived', maybe just call it Decision Posts? I like the picture!

File: index.qmd [hunk 1]
```diff
@@ -1,22 +1,50 @@
 ---
 title: Archive of Decisions Records
+about:
+  id: hero-heading
+  template: jolla
+  image: home.svg
+  image-title: "Making decisions"
+  image-alt: "Cartoon illustration of a man being confused and having to think about what decision to between between two arrows pointing in opposite directions."
```
Comment (signekb) [inline index.qmd:8]:
```suggestion
  image-alt: "A cartoon illustration of a person with their arms outstretched looking uncertain, with a green arrow pointing left and a black arrow pointing right, symbolising a choice.""
```

**Review by signekb — CHANGES_REQUESTED** [review]
Nice! I still like that the person in the illustration looks a bit confused 😁 
Just added a suggestion for the alt-text.

**Comment by lwjohnst86** [comment]
@K-Beicher Archive in the sense of these first three definitions: https://www.dictionary.com/browse/archive

>  1. Usually archives. documents or records relating to the activities, business dealings, etc., of a person, family, corporation, association, community, or nation.
> 2. archives, a place where public records or other historical documents are kept.
> 3. any extensive record or collection of data:

File: index.qmd [hunk 2]
```diff
@@ -1,22 +1,50 @@
 ---
 title: Archive of Decisions Records
```
Comment (lwjohnst86) [inline index.qmd:2]:
```suggestion
title: Archive of Decision Records
```

**Pull request merged**